### PR TITLE
[FIX] purchase_work_acceptance: fix view_move_form view

### PR DESCRIPTION
**Pull request by albariera**

### CODE REVIEW

File: purchase_work_acceptance/views/account_move_views.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <record id="view_move_form" model="ir.ui.view">
         <field name="name">account.move.form</field>
         <field name="model">account.move</field>
-        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="inherit_id" ref="purchase_stock.stock_move_purchase" />
```
Comment (etobella) [inline purchase_work_acceptance/views/account_move_views.xml]: maybe it would be better to use `purchase.view_move_form_inherit_purchase`

**Review by etobella — CHANGES_REQUESTED** [review]

File: purchase_work_acceptance/views/account_move_views.xml (hunk 1 above)
Comment (albariera) [reply purchase_work_acceptance/views/account_move_views.xml]: You are right! Thanks!

**Comment by kittiu** [comment]
@albariera what is the benefit of this vs the old one?
Thanks!

**Comment by albariera** [comment]
@kittiu There was a bug because it was trying to get the field "purchase_vendor_bill_id" from the "account.view_move_form" view, but it was not present in that view. This field is defined at "view_move_form_inherit_purchase" from the module "purchase". Thanks!

**Review by kittiu — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Comment by etobella** [comment]
> @kittiu There was a bug because it was trying to get the field "purchase_vendor_bill_id" from the "account.view_move_form" view, but it was not present in that view. This field is defined at "view_move_form_inherit_purchase" from the module "purchase". Thanks!

Also, an error was raised if a user without access to purchase accessed an account.move.

**Comment by kittiu** [comment]
Should this port to v14 as well?

**Comment by albariera** [comment]
Done @kittiu! #1282

**Comment by kittiu** [comment]
> Done @kittiu! #1282

🙏

**Pull request merged**